# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vmware/vmware-install.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- vmware-distrib/vmware-install.pl
+++ vmware-distrib/vmware-install.pl
1955,1956c1955,1959
<   $answer = get_persistent_answer('In which directory do you want to install the binary files?',
<                                   'BINDIR', 'dirpath', $rootdir . '/bin');
---
>   # $answer = get_persistent_answer('In which directory do you want to install the binary files?',
>   #     
>   #                                'BINDIR', 'dirpath', $rootdir . '/bin');
>   $answer = $rootdir.'/bin';
>   db_add_answer('BINDIR',$answer);
1983,1984c1986,1988
<     $answer = get_answer('In which directory do you want to install '
<                          . 'the daemon files?', 'dirpath', $rootdir . '/sbin');
---
> 	  #$answer = get_answer('In which directory do you want to install '
> 	  #               . 'the daemon files?', 'dirpath', $rootdir . '/sbin');
>     $answer = $rootdir . '/sbin';
1992,1993c1996,1998
<   $answer = get_answer('In which directory do you want to install the library files?',
<                        'dirpath', $rootdir . '/lib/vmware');
---
>   #$answer = get_answer('In which directory do you want to install the library files?',
>   #                     'dirpath', $rootdir . '/lib/vmware');
>   $answer = $rootdir . '/lib/vmware';
2007,2008c2012,2014
<     $answer = get_answer('In which directory do you want to install the manual files?',
<                          'dirpath', $mandir);
---
>     #$answer = get_answer('In which directory do you want to install the manual files?',
>     #                     'dirpath', $mandir);
>     $answer = $mandir;
2018,2019c2024,2026
<   $answer = get_answer('In which directory do you want to install the documentation files?',
<                        'dirpath', $docdir . '/vmware');
---
>   #$answer = get_answer('In which directory do you want to install the documentation files?',
>   #                     'dirpath', $docdir . '/vmware');
>   $answer = $docdir;
2056,2059c2063,2068
<   $answer = get_persistent_answer('What is the directory that contains the init'
<                                   .' directories (rc0.d/ to rc6.d/)?'
<                                   , 'INITDIR', 'initdirpath', $initdir);
< 
---
>   #$answer = get_persistent_answer('What is the directory that contains the init'
>   #                                .' directories (rc0.d/ to rc6.d/)?'
>   #                                , 'INITDIR', 'initdirpath', $initdir);
>   db_add_answer('INITDIR',$initdir);
>   $answer = $initdir;
>   
2069,2071c2078,2082
<   $answer = get_persistent_answer('What is the directory that contains the init'
<                                   .' scripts?', 'INITSCRIPTSDIR'
<                                   , 'initscriptsdirpath', $initscriptsdir);
---
>   #$answer = get_persistent_answer('What is the directory that contains the init'
>   #                                .' scripts?', 'INITSCRIPTSDIR'
>   #                                , 'initscriptsdirpath', $initscriptsdir);
>   $answer = $answer . '/init.d';
>   db_add_answer('INITSCRIPTSDIR',$answer);
2783c2794,2795
<   $ENV{'PATH'} = '/bin:/usr/bin:/sbin:/usr/sbin';
---
>   # --- removed for SDE
>   # $ENV{'PATH'} = '/bin:/usr/bin:/sbin:/usr/sbin';
2878,2880c2890,2894
<     db_remove_answer('EULA_AGREED');
<     db_remove_answer('ISC_COPYRIGHT_SEEN');
< 
---
>     #db_remove_answer('EULA_AGREED');
>     #db_remove_answer('ISC_COPYRIGHT_SEEN');
>     db_add_answer('EULA_AGREED','yes');
>     db_add_answer('ISC_COPYRIGHT_SEEN','yes');
>     
2888,2897c2902,2911
<         $answer = get_persistent_answer('Before running '
<                                         . vmware_product_name()
<                                         . ' for the first time, you need to '
<                                         . 'configure it by invoking the '
<                                         . 'following command: "'
<                                         . db_get_answer('BINDIR')
<                                         . '/' . "$gConfigurator" . '". Do you '
<                                         . 'want this program to invoke the '
<                                         . 'command for you now?'
<                                         , 'RUN_CONFIGURATOR', 'yesno', 'yes');
---
> 	      # $answer = get_persistent_answer('Before running '
> 	      #                          . vmware_product_name()
> 	      #                          . ' for the first time, you need to '
> 	      #                          . 'configure it by invoking the '
> 	      #                          . 'following command: "'
> 	      #                          . db_get_answer('BINDIR')
> 	      #                          . '/' . "$gConfigurator" . '". Do you '
> 	      #                          . 'want this program to invoke the '
> 	      #                          . 'command for you now?'
> 	      #                          , 'RUN_CONFIGURATOR', 'yesno', 'yes');
2899,2902c2913,2916
<         print wrap('Before running ' . vmware_product_name() . ' for the first'
<                    . ' time, you need to configure it by invoking the'
<                    . ' following command: "' . db_get_answer('BINDIR')
<                    . '/' . "$gConfigurator" . '"' . "\n\n", 0);
---
> 	      # print wrap('Before running ' . vmware_product_name() . ' for the first'
> 	      #     . ' time, you need to configure it by invoking the'
> 	      #     . ' following command: "' . db_get_answer('BINDIR')
> 	      #     . '/' . "$gConfigurator" . '"' . "\n\n", 0);
2908c2922
< 
---
>     $answer = 'no';
